Fix:

error: implicitly declaring library function 'strncpy' with type 'char *(char *, const char *, unsigned long)'

error: implicit declaration of function 'open' is invalid in C99

error: implicit declaration of function 'close' is invalid in C99

error: implicit declaration of function 'read' is invalid in C99

error: implicit declaration of function 'write' is invalid in C99
--- sys/grfileio.c.orig	1995-06-19 15:41:42.000000000 -0500
+++ sys/grfileio.c	2020-10-18 18:41:24.000000000 -0500
@@ -19,6 +19,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <sys/types.h>
+#include <string.h>
+#include <unistd.h>
 #include <fcntl.h>
 
 #ifdef PG_PPU
--- sys/grtermio.c.orig	1994-12-29 14:00:33.000000000 -0600
+++ sys/grtermio.c	2020-10-18 18:41:27.000000000 -0500
@@ -7,6 +7,9 @@
 
 #include <stdio.h>
 #include <termios.h>
+#include <string.h>
+#include <fcntl.h>
+#include <unistd.h>
 
 #ifdef PG_PPU
 #define GROTER groter_
